tools/test_cases.F90

```diff
@@ -874,6 +874,7 @@ subroutine init_case(u,v,w,pt,delp,q,phis, ps,pe,peln,pk,pkz,  uc,vc, ua,va, ak,
       call mp_stop()
       stop
       case(0)
+         UBar = 0.
```

Comment:
Wouldn't it be best to define Ubar = 0. (and the same for Vbar) at the initialization of the variable? Or does that bring up other compiler problems?